src/pages/_index/_components/what-is-wasmcloud/index.tsx

```diff
@@ -0,0 +1,194 @@
+import React from 'react';
+import { Section } from '@site/src/pages/_components/section';
+import { SectionTag } from '@site/src/pages/_components/section-tag';
+import { SectionHeading } from '@site/src/pages/_components/section-heading';
+import {
+  Switcher,
+  SwitcherButton,
+  SwitcherContent,
+  SwitcherList,
+} from '../what-is-wasmcloud/switcher';
+import { SectionSubheading } from '@site/src/pages/_components/section-subheading';
+import styles from './what-is-wasmcloud.module.css';
+
+type Props = {};
+
+const INTRO_CONTENT: {
+  tag: React.ReactNode;
+  heading: React.ReactNode;
+  description: React.ReactNode;
+} = {
+  tag: 'What is wasmCloud?',
+  heading: 'Fast-track the development, deployment, and management of distributed applications',
+  description: (
+    <p>
+      wasmCloud is an open source project from the Cloud Native Computing Foundation (CNCF) that
+      enables teams to build polyglot applications composed of reusable code and run
+      them—resiliently and efficiently—across any Kubernetes cluster, cloud, datacenter, or edge.
+    </p>
+  ),
+};
+
+const SWITCHER_CONTENT: Array<{
+  id: string;
+  title: React.ReactNode;
+  image: string;
+  features: Array<{
+    title: React.ReactNode;
+    description: React.ReactNode;
+    link?: string;
+    linkText?: React.ReactNode;
+  }>;
+}> = [
+  {
+    id: 'build',
+    image: '/img/pages/home/what-is-wasmcloud/build.svg',
+    title: 'Build',
+    features: [
+      {
+        title: 'Integrate with your existing ecosystem',
+        description: (
+          <>
+            wasmCloud has first-tier support for Kubernetes, VS Code, ArgoCD, Backstage, Chainguard,
+            and more.
+          </>
+        ),
+      },
+      {
+        title: 'Build Reusable Components',
+        description: (
+          <>
+            Using reusable components means that platform teams can build features that can be
+            leveraged across your entire organization.
+          </>
+        ),
+      },
+      {
+        title: 'Bring-your-own language(s) for polyglot apps',
+        description: (
+          <>
+            Build and run applications from components written in C, C#, C++, Rust, Python, Go, C#,
+            TypeScript and more.
+          </>
+        ),
+      },
+    ],
+  },
+  {
+    id: 'compose',
+    image: '/img/pages/home/what-is-wasmcloud/compose.svg',
```

Comment:
this image has a glitch on the right side for runtime
<img width="620" alt="image" src="https://github.com/user-attachments/assets/7187a9a6-5aaf-4f60-8725-1b35d44f1ab2">